site/content/en/docs/Advanced/scheduling-and-autoscaling.md

```diff
@@ -97,6 +97,12 @@ This affects the Cluster autoscaler, Allocation Scheduling, Pod Scheduling and F
 
 #### Cluster Autoscaler
 
+{{< alert title="SafeToEvict Feature Gate" color="info" >}}
```

Comment:
I think this should be moved below the first paragraph in this section (and we will need to rewrite this more once the feature gate moves to beta). The first paragraph is still true and will remain the default behavior. What we will change is the second paragraph to tell folks to use safetoevict instead of manually adding in the annotation.